web/src/common/ErrorBoundary.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, ErrorInfo, ReactNode } from 'react'
+import { Typography, Icon } from '@equinor/eds-core-react'
+import { account_circle } from '@equinor/eds-icons'
+
+interface Props {
+  children?: ReactNode
+}
+
+interface State {
+  hasError: boolean
+}
+
+class ErrorBoundary extends Component<Props, State> {
+  public state: State = {
+    hasError: true,
+  }
+
+  public static getDerivedStateFromError(_: Error): State {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true }
+  }
+
+  public componentDidCatch(error: Error, errorInfo: ErrorInfo) {
+    console.error('Uncaught error:', error, errorInfo)
+  }
+
+  public render() {
+    if (this.state.hasError) {
+      return (
+        <>
+          <Typography variant={'h3'}>Ops... Something went wrong 😞</Typography>
+          <Typography varant={'body_short'}>
+            You could try resetting the application by clicking the{' '}
+            <Icon data={account_circle} /> icon, and then &quot;reset
+            application data&quot;.
+          </Typography>
+          <Typography varant={'body_short'}>
+            This will cause you to lose application data. If that is not a good
+            option you should contact support (team-hermes@equinor.com).
+          </Typography>
```

Comment:
Sorry, I know I said this was good, but now I am not so sure. Many times, an error may be triggered by some odd combination of commands that is unlikely to happen twice in a row (ex: selecting a template, then deselecting it again, this caused the app to crash earlier - there are no reason why anyone would do that twice). The first thing people should do if something fails is therefore simply to reload the page and try again. The text as it is may cause people to lose data when they don't really have to.

Reply:
Rewrote into a numbered list, where 1. is "Reload page"